Libraries/Network/RCTNetworking.m

```diff
@@ -206,6 +206,7 @@ - (RCTURLRequestCancellationBlock)buildRequest:(NSDictionary<NSString *, id> *)q
   NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
   request.HTTPMethod = [RCTConvert NSString:RCTNilIfNull(query[@"method"])].uppercaseString ?: @"GET";
   request.allHTTPHeaderFields = [RCTConvert NSDictionary:query[@"headers"]];
+  request.timeoutInterval = [RCTConvert float:query[@"timeout"]] / 1000;
```

Comment:
This can be written as
    
    request.timeoutInterval = [RCTConvert NSTimeInterval:query[@"timeout"]];

(The NSTimeInterval: converter already does the conversion from ms to seconds)